### PR TITLE
chore: add aws kms to bdd test

### LINKED ISSUE #811: Add aws kms support

### PR DESCRIPTION
**Pull request by fqutishat**
- Add tests for P-256, P-384 and P-256k1

closes #811

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by aholovko — APPROVED** [review]

File: test/bdd/fixtures/docker-compose.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@ services:
       - DATABASE_TYPE=mongodb
       - DATABASE_URL=mongodb://mongodb.example.com:27017
       - DATABASE_PREFIX=vc_rest_echo_
-      - VC_REST_DEFAULT_KMS_TYPE=local
```
[the remark this answers is not in the document]
Comment (fqutishat) [reply test/bdd/fixtures/docker-compose.yml:19]: I will remove it in next PR

**Review by vkubiv — APPROVED** [review]

File: test/bdd/fixtures/docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rolsonquadras) [reply test/bdd/fixtures/docker-compose.yml:19]: Is this change specific to aws https://github.com/trustbloc/vcs/pull/812/files#diff-02b497cd33b5daa3c7bbe1bd79c09be0156b72503fc7e67aed41ad6f7a6f57c5L130 ? Does local kms break with this change ?

File: test/bdd/pkg/v1/issuer/issuer_steps.go [hunk 2]
```diff
@@ -83,8 +83,8 @@ func (e *Steps) createIssuerProfile(user, organizationName string) error { //nol
 			Contexts:         nil,
 			DidMethod:        "orb",
 			Format:           "ldp_vc",
-			KeyType:          "ED25519",
-			SigningAlgorithm: "Ed25519Signature2018",
+			KeyType:          "ECDSAP256DER",
```
Comment (rolsonquadras) [inline test/bdd/pkg/v1/issuer/issuer_steps.go:86]: @aholovko @sudeshrshetty Is there a list of supported key types ? I don't see it in OpenAPI spec (ideally we should have an enum).

Comment (aholovko) [reply test/bdd/pkg/v1/issuer/issuer_steps.go:86]: I can see that we have validation for supported key types depending on concrete key manager (https://github.com/trustbloc/vcs/blob/main/pkg/restapi/v1/issuer/controller.go#L324), so not sure if we can just enumerate supported key types in spec. cc @vkubiv ^

Comment (vkubiv) [reply test/bdd/pkg/v1/issuer/issuer_steps.go:86]: yes they depend on kms, also they depend on signatureAlgorithm

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**